src/hooks/useAffiliatesInfo.ts

```diff
@@ -37,13 +37,13 @@ export const useAffiliatesInfo = (dydxAddress?: string) => {
 
     try {
       const [metaDataResponse, totalVolumeResponse, affiliateInfo] = await Promise.all([
-        safeFetch(`${metadataEndpoint}?address=${encodeURIComponent(dydxAddress)}`, {
+        fetch(`${metadataEndpoint}?address=${encodeURIComponent(dydxAddress)}`, {
```

Comment:
cc @tyleroooo reverting to regular fetch here because we want 404 to be considered valid response (if user doesnt have affiliate stats yet)